app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/Project.scala

```diff
@@ -63,6 +73,9 @@ object Project extends GeoJsonSupport {
         description,
         visibility,
         tileVisibility,
+        isAOIProject,
+        aoiCadence,
+        now,
```

Comment:
I opt to avoid `null` at all costs, as it undermines type safety. I like `now - aoiCadence`.